DBM-Naxx/MilitaryQuarter/Horsemen.lua

```diff
@@ -49,3 +92,34 @@ function mod:SPELL_AURA_APPLIED_DOSE(args)
 	end
 end
 
+function mod:SPELL_SUMMON(args)
+	if args:IsSpellName("Void Zone") then
```

Comment:
Wouldn't this better be handled with spell IDs?

Currently it will break in non enUS/enGB clients.

Reply:
Ye but then you will require to scan everyfight with incident since the id's can be changed depending on server, and the names always stays the same

Reply:
Plus i don't get it, boss emotes on private servers always stay in english, but boss abilities don't? Where's the logic in that?!